app/workers/regenerate_image_worker.rb

```diff
@@ -0,0 +1,26 @@
+class RegenerateImageWorker
+  include Sidekiq::Worker
+
+  def perform doc_ids
+    documents = Document.find(doc_ids)
+    documents.each { |doc| doc.update! processed: false } unless documents.blank?
+    processing = false
+    current = doc_ids.pop
+    while !doc_ids.blank?
+      if processing
+        if Document.find(current).processed
+          Rails.logger.info "Regenerated images for document with id #{current},
```

Comment:
Could this logic go into the PdfToImagesWorker? My general thought here would be that if the PdfToImagesWorker returns whatever its successful condition is then we could print this to the log from that worker.